rust/src/coordinator.rs

```diff
@@ -45,9 +80,12 @@ pub struct Coordinator {
     phase: Phase,
 
     // round dictionaries
-    dict_sum: HashMap<box_::PublicKey, box_::PublicKey>,
-    dict_seed: HashMap<box_::PublicKey, HashMap<box_::PublicKey, Vec<u8>>>,
-    dict_mask: Counter<Vec<u8>>,
+    /// Dictionary built during the sum phase.
+    dict_sum: SumDict,
+    /// Dictionary built during the update phase.
+    dict_seed: SeedDict,
+    /// Dictionary built during the sum2 phase.
```

Comment:
is this naming right? is a `Counter` a "dictionary"?

Reply:
it is very closely related: https://docs.rs/counter/0.4.3/counter/#treat-it-like-a-map, basically behaves like a hashmap from `T` to `int` with some added functionality.